fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/ProductAnalysis.kt

```diff
@@ -303,6 +311,91 @@ private fun SettingsCard(
     }
 }
 
+@Composable
+private fun ReviewQualityInfoCard(
+    onLearnMoreClick: () -> Unit,
+) {
+    ReviewQualityCheckExpandableCard(
+        modifier = Modifier.fillMaxSize(),
+        title = stringResource(id = R.string.review_quality_check_info_title),
+    ) {
+        Column(
+            verticalArrangement = Arrangement.spacedBy(24.dp),
+        ) {
+            // Any and all text formatting (bullets, inline substring bolding, etc.) will be handled as
+            // follow-up when the copy is finalized.
+            Text(
+                text = stringResource(id = R.string.review_quality_check_info_overview),
+                style = FirefoxTheme.typography.body2,
+            )
+
+            val link = stringResource(R.string.review_quality_check_info_learn_more_link)
+            val text = stringResource(R.string.review_quality_check_info_learn_more, link)
+            val linkStartIndex = text.indexOf(link)
+            val linkEndIndex = linkStartIndex + link.length
+            ClickableSubstringLink(
+                text = text,
+                textStyle = FirefoxTheme.typography.body2,
+                clickableStartIndex = linkStartIndex,
+                clickableEndIndex = linkEndIndex,
+                onClick = onLearnMoreClick,
+            )
+
+            Text(
+                text = stringResource(id = R.string.review_quality_check_info_review_grade_header),
+                style = FirefoxTheme.typography.body2,
+            )
+
+            ReviewGradingScaleInfo(
+                reviewGrades = listOf(
+                    ReviewQualityCheckState.Grade.A,
+                    ReviewQualityCheckState.Grade.B,
+                ),
+                info = stringResource(id = R.string.review_quality_check_info_grade_info_AB),
+            )
+
+            ReviewGradingScaleInfo(
+                reviewGrades = listOf(ReviewQualityCheckState.Grade.C),
+                info = stringResource(id = R.string.review_quality_check_info_grade_info_C),
+            )
+
+            ReviewGradingScaleInfo(
+                reviewGrades = listOf(
+                    ReviewQualityCheckState.Grade.D,
+                    ReviewQualityCheckState.Grade.F,
+                ),
+                info = stringResource(id = R.string.review_quality_check_info_grade_info_DF),
+            )
+        }
+    }
+}
+
+@Composable
+private fun ReviewGradingScaleInfo(
+    reviewGrades: List<ReviewQualityCheckState.Grade>,
+    info: String,
+) {
+    Row(
+        modifier = Modifier.fillMaxWidth(),
+        verticalAlignment = Alignment.Top,
+        horizontalArrangement = Arrangement.spacedBy(8.dp),
+    ) {
+        reviewGrades.forEach { grade ->
+            ReviewGradeCompact(grade = grade)
+        }
+
+        if (reviewGrades.size == 1) {
```

Comment:
❓ Curious why this was chosen over setting the Row width to 64.dp?

Reply:
so this was primarily me just exploring an alternative approach. I honestly wasn't 100% how the 64 in your patch was calculated, so I opted for using the conditional spacer with 24, since that's the size of the letter grades (which we could then refactor into a dimen).

how does that sound?

Reply:
It was basically (size of letter grade + spacer) * 2.

I think it's fine either way, only minor concern was with the if logic, but even 64 is kind of an implicit logic which will break if the size of letter grade changes. Here we are being explicit, so this is better. 👍 

The dimens approach sounds good, I did try to introduce that early on but we need more ux sync there.